kafka/src/main/java/zipkin2/reporter/kafka/KafkaSender.java

```diff
@@ -306,7 +306,7 @@ AdminClient getAdminClient() {
     if (producer != null) producer.close();
     AdminClient adminClient = this.adminClient;
     // Intentionally using deprecated method to avoid pinning newer Kafka client
-    if (adminClient != null) adminClient.close(1, TimeUnit.SECONDS);
+    if (adminClient != null) adminClient.close();
```

Comment:
Update admin close to undefined timeout as method with timeunit is not available anymore.

Reply:
What's the minimum Kafka version supported now? We should probably document this somewhere.

Reply:
With this change we fallback to a close method that is supported across versions -- if we were to use `.close(Duration)`, then it will limit versions; also sniffer was complaining about Duration as it's JDK 8 AFAIR.

Reply:
I see. So we've kept library version compatibility, but now the behavior is changed - it will wait indefinitely for close now. I guess that's not such a big deal since we already wait indefinitely to close the `KafkaProducer`. Thanks for the explanation!

Reply:
I guess the comment on line 308 can be removed now, though, since we're no longer using a deprecated method.